<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="maximumSize">
       <size>
        <width>177</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>177</width>
         <height>446</height>
        </rect>
       </property>
       <attribute name="label">
        <string>流程管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnToDo">
          <property name="text">
           <string>我的待办</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSubmited">
          <property name="text">
           <string>我的流程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSubmited_2">
          <property name="text">
           <string>业务管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btTaskSheet">
          <property name="text">
           <string>任务单管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSamplingSchedule">
          <property name="text">
           <string>采样排单</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btTaskSheet_2">
          <property name="text">
           <string>采样任务管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btTaskSheet_3">
          <property name="text">
           <string>样品流转</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btTaskSheet_4">
          <property name="text">
           <string>分析任务管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reportManagerBtn">
          <property name="text">
           <string>报告管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btRMManage">
          <property name="text">
           <string>标准物质管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btRMManage_2">
          <property name="text">
           <string>设备管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btRMManage_3">
          <property name="text">
           <string>溯源管理</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>177</width>
         <height>446</height>
        </rect>
       </property>
       <attribute name="label">
        <string>系统工具</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btPersonalInfo">
          <property name="text">
           <string>资料修改</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>177</width>
         <height>446</height>
        </rect>
       </property>
       <attribute name="label">
        <string>管理工具</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btEmployeeManage">
          <property name="text">
           <string>人员管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btLabCapability">
          <property name="text">
           <string>检测能力管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btDBManage">
          <property name="text">
           <string>数据库管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btModInit">
          <property name="text">
           <string>模块初始化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="standardsManagerBtn">
          <property name="text">
           <string>标准管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>危废管理</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionlogin"/>
    <addaction name="exitAct"/>
    <addaction name="actionInitMod"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="skinMenu">
    <property name="title">
     <string>皮肤</string>
    </property>
    <addaction name="skin1Action"/>
    <addaction name="skin2Action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="skinMenu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogin">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionInitMod">
   <property name="text">
    <string>初始化模块</string>
   </property>
  </action>
  <action name="skin1Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>绿意</string>
   </property>
  </action>
  <action name="skin2Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>经典</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>版本信息</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btRMManage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpenTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEmployeeManage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpenTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDBManage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpenTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenTab()</slot>
 </slots>
</ui>
